lib/steep/server/type_check_worker.rb

```diff
@@ -74,9 +74,9 @@ def handle_request(request)
         when "textDocument/didChange"
           collect_changes(request)
 
-        when "textDocument/didOpen"
-          uri = request[:params][:textDocument][:uri]
-          text = request[:params][:textDocument][:text]
+        when "$/file/reset"
```

Comment:
Introduce new `$/file/reset` endpoint that resets the content of a file.
Used by `textDocument/didOpen` and `workspace/didChangeWatchedFiles`.